config/peerpods/podvm/README.md

```diff
@@ -34,3 +34,28 @@ Now when you create a KataConfig with `enablePeerPods: true` with empty
 `AZURE_IMAGE_ID` or `AWS_AMI_ID` in `peer-pods-cm`, then depending on the cloud
 provider configured, the operator will create the pod VM image based on the
 provided config.
+
+## PodVM image upload configuration
+
+The PodVM image can be embedded to a container image and used to unwrap and upload to the libvirt volume provided in the `peer-pods-secret`. 
+This feature is currently only supported for libvirt provider.
+
+To build an OCI image with PodVM image, `Dockerfile.podvm-oci` can be used as,
+```
+docker build -t podvm-libvirt \
+    --build-arg PODVM_IMAGE_SRC=<podvm_image_source> \
+    -f Dockerfile.podvm-oci .
+```
+Here the `PODVM_IMAGE_SRC` is where the `qcow2` image is present on the host and optionally `PODVM_IMAGE_PATH` can also be set which is the path of the qcow2 image inside the container image.
```

Comment:
nit, maybe worth mentioning that if `PODVM_IMAGE_PATH` is set it should be also specified as the image path

Reply:
sure, thanks for the pointer Snir, will add it..

Reply:
incorporated..